### DISCUSSION TITLE
Edge Runtime API only

### DISCUSSION THREAD

**Discussion by rreeves8** [vercel/edge-runtime#349] [Q&A]
I'm trying to make a rest api using the Edge network. I want to beable to import files for re-using code. Currently when I deploy the project, all my files become available either statically or as edge functions. 

I tried using Next Js however I'm having trouble with neon database and keysely during the build process. 

I dont want my other code files available statically. Is there a build process available from vercel where I can either wrap all my imported files into the single files? Or is there a config setting where I can disable static public access to the other files,  execpt the edge functions can still use them ? Using vecel ignore makes the imports unusable, there must be some specfic config I can use to make just the edge functions accessible publicly

**Comment by Kikobeats** [comment]
Hello,

I think you need some way to generate the Edge Functions markup and write them as files.

That makes sense to you?
